data_handling/models/comments.models.js

```diff
@@ -0,0 +1,19 @@
+const db = require('../../db/connection');
+
+
+
+exports.fetchCommentsByArticleId = (id) => {
+    
+    let query = "SELECT * FROM comments WHERE article_id=$1 ORDER BY created_at DESC"
+
+    return db.query(query,[id]).then((result) => {
+        if (result.rows.length === 0) {
```

Comment:
If there aren't any comments for an article but the article exists, this should be a 200, but what comes back from the model will be nothing. To distinguish between these two I would make another call to the database to see if the article does exist, if it does let this through as a valid response